### DISCUSSION TITLE
Parquet logical type may be none even if converted type is set?

### DISCUSSION THREAD

**Discussion by jp0317** [apache/arrow-rs#6404] [General]
when converted type is set but logical type is not, currently the [logical type](https://github.com/apache/arrow-rs/blob/master/parquet/src/schema/types.rs#L631) in basic type info will be none. I noticed that in C++ implementation logical type can be created from converted type, e.g., [here](https://github.com/apache/arrow/blob/main/cpp/src/parquet/schema.cc#L226). Is this behavior inconsistency intended?

**Comment by Jefffrey** [comment]
I think this explains it

https://github.com/apache/arrow-rs/blob/12091af35e5070dae33e86fbe988d9319521cd16/parquet/src/basic.rs#L1197-L1203

Can't speak on the C++ implementation, though it looks like they do it like this

https://github.com/apache/arrow/blob/deed9c0d99b33be112f9931ec92f4df65c75fc43/cpp/src/parquet/types.cc#L462-L532
